frontend/src/pages/WorkloadDetails/WorkloadPodLogs.tsx

```diff
@@ -1279,7 +1293,8 @@ const formatDate = (timestamp: string): string => {
 const mapStateToProps = (state: KialiAppState): ReduxProps => {
   return {
     kiosk: state.globalState.kiosk,
-    timeRange: timeRangeSelector(state)
+    timeRange: timeRangeSelector(state),
+    tracingIntegration: state.tracingState.info ? state.tracingState.info.integration : false
```

Comment:
Shorter. Not tested but this should work.
```suggestion
    tracingIntegration: state.tracingState.info?.integration ?? false
```